virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -71,13 +93,19 @@ def get_non_root_disk_names(session, ignore_status=False):
     if s:
         raise exceptions.TestError("Couldn't list block devices: '%s'" % o)
     LOG.debug("lsblk output:\n%s", o)
+
+    # search for additional root disk in case of image_mode
+    root_mountpoints = ["/"]
+    if is_image_mode(session):
+        root_mountpoints.append("/sysroot")
+
     lines = o.split("\n").copy()
     root_disk = None
     root_mounted = False
     disk_pattern = re.compile(r"^([a-z]+|sr\d)[\s\t$]")
     entry_pattern = re.compile(r"(.*)[\s\t]+(.*)")
     idx = 0
-    while -1 < idx < len(lines) + 1:
```

Comment:
Oh, great catch! So, that happened when there was no "/" and led to 

```
2025-07-28 10:28:05,523 avocado.test stacktrace       L0049 ERROR|   File "/var/ci/libvirt-ci/runtest/avocado-vt/avocado-vt/virttest/utils_libvirt/libvirt_disk.py", line 81, in get_non_root_disk_names
2025-07-28 10:28:05,523 avocado.test stacktrace       L0049 ERROR|     line = lines[idx]
2025-07-28 10:28:05,523 avocado.test stacktrace       L0049 ERROR| IndexError: list index out of range
```

Glad you found it.